universal_deep_transformer/distributed/TrainingDistributedUDT.ipynb

```diff
@@ -114,16 +116,16 @@
     "    model = get_udt_model()\n",
     "    model = dist.prepare_model(model)\n",
     "\n",
-    "    metrics = model.train_distributed_v2(\n",
-    "        filename=os.path.join(config[\"curr_dir\"], train_filenames[session.get_world_rank()]),\n",
+    "    metrics = model.train_distributed(\n",
+    "        filename=os.path.join(config[\"curr_dir\"], train_filenames[train.get_context().get_world_rank()]),\n",
     "        learning_rate=0.02,\n",
     "        epochs=1,\n",
     "        batch_size=256,\n",
     "        metrics=[\"categorical_accuracy\"],\n",
     "        verbose=True,\n",
     "    )\n",
     "\n",
-    "    session.report(\n",
+    "    train.report(\n",
     "        metrics=metrics,\n",
     "        checkpoint=dist.UDTCheckPoint.from_model(model),\n",
```

Comment:
Don't we need to pass in RunConfig for this to work? Anyway, I would still think we should add RunConfig, since even though it works here for one machine, we should add something here saying that, you would need `RunConfig`.

Reply:
Added now.